### ISSUE TITLE
Frequency GUI on QIO importer/exporter doesn't open

### ISSUE THREAD

**Issue by JohnWalker-117** [mekanism/Mekanism#7416]
### Issue description

Sometimes it just stops opening on all worlds all of sudden and I don't know why nor how to reproduce it, does not crash the game but prevents from using only those two, all other frequencies(QE, Drives, etc...) work as usual.

### Minecraft version

1.16.5 (Latest)

### Forge version

36.2.28

### Mekanism version

10.1.1 (Latest)

### Other relevant versions

MODLIST:
"[1.16.5][Forge]+RPG-HUD-3.8.4.jar"
"alexsmobs-1.12.1.jar"
"AppleSkin-mc1.16.2-forge-1.0.14.jar"
"Aquaculture-1.16.5-2.1.21.jar"
"architectury-1.17.16-forge.jar"
"AzureDoom-Chunk-Loader-1.16.4-1.0.0.jar"
"BasicNetherOres-1.16.5-5.5.0.jar"
"BetterCaves-Forge-1.16.4-1.1.2.jar"
"BetterMineshafts-Forge-1.16.4-2.0.3.jar"
"Bookshelf-1.16.5-10.0.8.jar"
"buildinggadgets-1.16.5-3.8.2.jar"
"byg-1.3.5.jar"
"catalogue-1.5.0-1.16.5.jar"
"chunkloaders-1.1.10-mc1.16.jar"
"citadel-1.8.1-1.16.5.jar"
"ClickMachine-1.16.4-4.3.0.jar"
"cloth-config-4.11.26-forge.jar"
"Clumps-6.0.0.24.jar"
"cofh_core-1.16.5-1.4.2.9.jar"
"collective-1.16.5-4.0.jar"
"comforts-forge-1.16.5-4.0.0.4.jar"
"configured-1.5.1-1.16.5.jar"
"constructionwand-1.16.5-2.2.jar"
"Controlling-7.0.0.20.jar"
"CookingForBlockheads_1.16.5-9.3.1.jar"
"createaddition-1.16.5-20220129a.jar"
"create-mc1.16.5_v0.3.2g.jar"
"Cucumber-1.16.5-4.1.12.jar"
"curioofundying-forge-1.16.5-5.2.0.0.jar"
"curios-forge-1.16.5-4.0.5.1.jar"
"DarkUtilities-1.16.5-8.0.9.jar"
"DungeonsArise-1.16.5-2.1.47-release.jar"
"dynamiclightsreforged-mc1.16.5_v1.0.1.jar"
"easy_villagers-1.16.4-1.1.28.jar"
"EnchantmentDescriptions-1.16.5-7.0.8.jar"
"essentials-1.16.5-2.11.0.jar"
"ExtremeReactors2-1.16.5-2.0.38.jar"
"flywheel-1.16-0.2.5.jar"
"ForgeEndertech-1.16.4-7.1.0.0-build.0075.jar"
"FpsReducer-mc1.16.5-1.19.jar"
"ftb-backups-2.1.1.6.jar"
"GameMenuModOption-1.16.4-1.8.jar"
"geckolib-forge-1.16.5-3.0.46.jar"
"gems-1.16.5-1.8.0.jar"
"guardscraft-1.16.5-1.1.0Beta.jar"
"guardvillagers-1.16.5.1.2.6.jar"
"ImmersiveEngineering-1.16.5-5.0.7-143.jar"
"ImmersivePetroleum-1.16.5-3.3.0-11.jar"
"InventoryHud_[1.16.2-1.16.5].forge-3.3.0.jar"
"invtweaks-1.16.4-1.0.1.jar"
"item-filters-forge-1605.2.5-build.9.jar"
"jei-1.16.5-7.7.1.145.jar"
"jeiintegration_1.16.5-7.0.1.15.jar"
"JustEnoughProfessions-1.16.5-1.2.2.jar"
"JustEnoughResources-1.16.5-0.12.1.114.jar"
"light-overlay-5.8.1.jar"
"littlelogistics-mc1.16.5-v1.0.2.jar"
"magnesium_extras-mc1.16.5_v1.3.1.jar"
"magnesium-1.5.jar"
"Mantle-1.16.5-1.6.103.jar"
"Mekanism-1.16.5-10.1.1.456.jar"
"MekanismAdditions-1.16.5-10.1.1.456.jar"
"MekanismGenerators-1.16.5-10.1.1.456.jar"
"MekanismTools-1.16.5-10.1.1.456.jar"
"minecolonies-1.16.5-1.0.429-RELEASE.jar"
"MouseTweaks-2.14-mc1.16.2.jar"
"MysticalAgradditions-1.16.5-4.2.4.jar"
"MysticalAgriculture-1.16.5-4.2.6.jar"
"Ore_Portal_V1.11_hard.jar"
"overloadedarmorbar-5.1.0.jar"
"PackingTape-1.16.3-0.10.0.jar"
"Patchouli-1.16.4-51.jar"
"performant-1.16.2-5-3.85m.jar"
"Placebo-1.16.4-4.4.5.jar"
"pneumaticcraft-repressurized-1.16.5-2.12.5-190.jar"
"polymorph-forge-1.16.5-0.24.jar"
"productivebees-1.16.5-0.6.9.25.jar"
"Quick+Harvest-1.16.4-1.2.0.jar"
"rangedwirelessredstone-1.16.5-0.1.4.jar"
"ropebridge-1.16.4-2.1.jar"
"rsgauges-1.16.4-1.2.10.jar"
"SaveYourPets-1.16.2-1.0.0.5.jar"
"shutupexperimentalsettings-1.0.3.jar"
"simplefarming-1.16.5-1.3.6.jar"
"smallships-1.16.5-1.10.1.jar"
"smoothboot-forge-1.16.4-1.2.2.jar"
"stoneholm-1.2.2.jar"
"StorageDrawers-1.16.3-8.2.2.jar"
"structurize-1.16.5-1.0.343-RELEASE.jar"
"supermartijn642configlib-1.0.9-mc1.16.jar"
"supermartijn642corelib-1.0.16-mc1.16.5.jar"
"supplementaries-1.16.5-0.13.0.jar"
"theoneprobe-1.16-3.1.4.jar"
"thermal_cultivation-1.16.5-1.4.1.3.jar"
"thermal_expansion-1.16.5-1.4.2.4.jar"
"thermal_foundation-1.16.5-1.4.3.10.jar"
"thermal_innovation-1.16.5-1.4.1.3.jar"
"thermal_locomotion-1.16.5-1.4.1.2.jar"
"tinygates-0.1.2a.jar"
"tinyredstone-1.16.5-1.13.3.jar"
"titanium-1.16.5-3.2.8.2.jar"
"tl_skin_cape_forge_1.16.5-1.19.jar"
"torcherino-13.0.3+1.16.3.jar"
"torchmaster-2.3.8.jar"
"treeharvester_1.16.5-4.1.jar"
"valhelsia_core-16.0.8.jar"
"valhelsia_structures-1.16.5-0.1.1.jar"
"Wither-Skeleton-Tweaks-1.16.4-5.3.0.jar"
"Xaeros_Minimap_21.8.2_Forge_1.16.5.jar"
"XaerosWorldMap_1.14.1_Forge_1.16.5.jar"
"YungsApi-1.16.4-Forge-6.jar"
"ZeroCore2-1.16.5-2.1.9.jar"

**Comment by pupnewfster** [comment]
Is there any errors printed to the log? (Either server or client side)

**Comment by JohnWalker-117** [comment]
> 
[2022-02-23-7.log](https://github.com/mekanism/Mekanism/files/8127189/2022-02-23-7.log)

This is the latest log file on the arquives

**Comment by pupnewfster** [comment]
The provided log looks like the game wasn't even able to finish launching (potentially due to performant or some other coremod it is a bit hard to tell). The latest log file should be called `latest.log` in the `logs` folder.

**Comment by JohnWalker-117** [comment]
[latest.log](https://github.com/mekanism/Mekanism/files/8127224/latest.log)

There it is, sorry I don't know much about mine logs

**Comment by JohnWalker-117** [comment]
Just adding here, now anything that requires clicking on a frequency selection(Portable Drives, QIO drives and digital miners) do not work

**Comment by FoxSGR** [comment]
The same is happening here. Only happens in a multiplayer server. Works fine in singleplayer.

Nothing appears in the console/logs in the client or server.

Tried with an brand new world and can still reproduce.

<details>
<summary>Mod list:</summary>

```
[1.16.5] SecurityCraft v1.9.0.2.jar
advancedperipherals-1.16.5-0.7.7.1r.jar
AI-Improvements-1.16.5-0.4.0.jar
alexsmobs-1.12.1.jar
appleskin-forge-mc1.16.x-2.4.0.jar
appliedenergistics2-8.4.7.jar
architectury-1.30.55.jar
AutoRegLib-1.6-49.jar
BetterAdvancements-1.16.5-0.1.0.108.jar
betteranimalsplus-1.16.5-11.0.5-forge.jar
BetterThirdPerson-Forge-1.16.4-1.5.1.jar
BiomesOPlenty-1.16.5-13.1.0.477-universal.jar
blockcarpentry-1.16-0.4.0.jar
Bookshelf-Forge-1.16.5-10.4.31.jar
BrandonsCore-1.16.5-3.0.13.244-universal.jar
buildinggadgets-1.16.5-3.8.2.jar
CBMultipart-1.16.5-3.0.4.123-universal.jar
cc-tweaked-1.16.5-1.100.4.jar
cgm-1.1.0-1.16.5.jar
ChickenChunks-1.16.5-2.9.1.89-universal.jar
Chisel-MC1.16.5-2.0.1-alpha.4.jar
chiselsandbits-1.0.63.jar
ChunkAnimator-1.16.5-1.2.4.jar
Chunk Pregenerator-V1.16-3.2.1.jar
citadel-1.8.1-1.16.5.jar
ClientTweaks_1.16.3-5.3.0.jar
cloth-config-4.14.54-forge.jar
Clumps-6.0.0.28.jar
CodeChickenLib-1.16.5-4.0.7.445-universal.jar
Controlling-7.0.0.28.jar
CTM-MC1.16.1-1.1.2.6.jar
DarkUtilities-1.16.5-8.0.11.jar
dcintegration-forge-2.3.5-1.16.jar
Draconic-Evolution-1.16.5-3.0.21.443-universal.jar
DynamicSurroundings-1.16.5-4.0.5.0.jar
Electrodynamics-1.16.5-0.5.0-2.jar
elevatorid-1.16.5-1.7.13.jar
EnchantmentDescriptions-1.16.5-7.0.18.jar
envirocore-1.16.5-3.0.9.3.jar
enviromats-1.16.5-2.0.9.0.jar
envirotech-1.16.5-3.0.9.4.jar
FallingTree-1.16.5-2.11.5.jar
FastLeafDecay-v25.2.jar
FastWorkbench-1.16.5-4.6.1.jar
ferritecore-2.0.7-forge.jar
FpsReducer-forge-1.24-mc1.16.5.jar
geckolib-forge-1.16.5-3.0.65.jar
gravestone-1.16.5-1.0.7.jar
Hwyla-forge-1.10.11-B78_1.16.2.jar
industrial-foregoing-1.16.5-3.2.14.6-14.jar
Intercontinental+Ballistic+Redux-1.16.5-1.4.0-4C1BF5124.jar
inventorypets-1.16.5-2.0.15.jar
invtweaks-1.16.4-1.0.1.jar
ironchest-1.16.5-11.2.21.jar
jei-1.16.5-7.7.1.152.jar
jeiintegration_1.16.5-7.0.1.15.jar
journeymap-1.16.5-5.7.3.jar
JustEnoughResources-1.16.5-0.12.1.133.jar
kotlinforforge-1.17.0-obf.jar
lazydfu-0.1.3.jar
light-overlay-5.8.1.jar
mcw-bridges-2.0.2-mc1.16.5.jar
mcw-doors-1.0.6-mc1.16.5.jar
mcw-fences-1.0.3-mc1.16.5.jar
mcw-lights-1.0.3-mc1.16.5.jar
mcw-roofs-2.1.1-mc1.16.5.jar
mcw-trapdoors-1.0.5-mc1.16.5.jar
mcw-windows-2.0.2-mc1.16.5.jar
Mekanism-1.16.5-10.1.2.457.jar
MekanismAdditions-1.16.5-10.1.2.457.jar
MekanismGenerators-1.16.5-10.1.2.457.jar
MekanismTools-1.16.5-10.1.2.457.jar
MouseTweaks-2.14-mc1.16.2.jar
mowziesmobs-1.5.22.jar
Neat 1.7-27.jar
NetherPortalFix_1.16.3-7.2.1.jar
Nuclear_Science-1.16.5-0.4.0-1.jar
obfuscate-0.6.2-1.16.3.jar
Patchouli-1.16.4-53.2.jar
Placebo-1.16.5-4.6.1.jar
Quark-r2.4-322.jar
randompatches-2.4.4-forge.jar
repurposed_structures_forge-3.4.7+1.16.5.jar
Runelic-1.16.5-7.0.2.jar
Scorge-3.1.3.jar
SERVERS_MoGuns-1.5-1.16.5.jar
snowundertrees-1.16.5-v1.3.jar
Space-BossTools-1.16.5-5.5e.jar
titanium-1.16.5-3.2.8.7-22.jar
valkyrielib-1.16.5-3.0.9.5.jar
```
</details>

**Comment by FoxSGR** [comment]
Copied the world from the multiplayer server to the client saves folder, opened it in singleplayer and the GUI does open there.

I tried removing most mods in common with @JohnWalker-117 but it still doesn't work.

**Comment by pupnewfster** [comment]
No idea if this is related but one weird thing I just stumbled on is that apparently the QIO frequency tab won't open if the security system is disabled (I will fix this for 1.18, 10.2.1). But if you look at the server's `config/Mekanism/general.toml` see if `allowProtection` is set to `true` or `false`, and if it is set to `false` for some reason change it to `true`. My guess given it is on a per dashboard basis from my understanding and not every QIO thing this is not the issue but figured it is worth checking.

**Comment by FoxSGR** [comment]
That was it! Changing `allowProtection` to `true` worked. Thank you!

We had disabled it because my friend switched accounts.

**Comment by pupnewfster** [comment]
Nice, this won't be fixed in 1.16 given the 1.16 branch is EOL but I will make sure to remove it checking that for some reason even though it doesn't have anything to do with security for the next 1.18 release.
